docs/Technical-Documentation/TDP-environments-README.md

```diff
@@ -0,0 +1,73 @@
+# TDP Environments
+
+## Development
+
+| Dev Site | Frontend URL | Backend URL | Purpose |
+| -------- | -------- | -------- | -------- |
+| Sandbox     | https://tdp-frontend-sandbox.app.cloud.gov | https://tdp-backend-sandbox.app.cloud.gov     | Space for devs to test in a deployed environment 
+| A11y | https://tdp-frontend-a11y.app.cloud.gov | https://tdp-backend-a11y.app.cloud.gov | Space for accessibility testing |
+| QASP | https://tdp-frontend-qasp.app.cloud.gov | https://tdp-backend-qasp.app.cloud.gov | Space for QASP review |
+| raft | https://tdp-frontend-raft.app.cloud.gov | https://tdp-backend-raft.app.cloud.gov | Space for raft review |
+
+### Dependencies 
+
+- `clamav-rest`
+
+**Cloud.gov AWS RDS `(tanf-dev)`**
+- `tdp-db-dev`
```

Comment:
will any parsed test data be stored here? If so, could include this in description of what's here?

Reply:
> not from unit tests - that data gets deleted on test teardown. this database stores users, groups, stts, metadata about data files that have been submitted, etc (Django data models). It has not been decided if parsed data will end up in this DB or another service

Reply:
Test data is not stored here; this DB will primarily contain the Users, Reports, and other application-related models.